### DISCUSSION TITLE
Identify and mark unique files?

### DISCUSSION THREAD

**Discussion by kdpdev1** [hsoft/dupeguru#1079] [Q&A]
Is it possible to run DupeGuru on two drives and get a report of the files that are unique to each drive? The use case is that the drives are nearly the same as one another, but I want to identify the areas where they are different.

**Comment by butter-fly** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by digitaldivergence** [comment]
I had the same question as kdpdev1 ... reason for this is I have a colleague that duplicates files and folders everywhere (he is 80+)  - So i can run dedupe but move duplicates but I have built a reference folder area - he can have 20 copies and also has a terrible naming policy.  If I could do an inverse of find duplicates - but find unique - then I can zoom in on the files I am missing .... why I need this - he has folders with 250 000 files .... and 20 of these ... made worse by he sometimes deletes the wrong folders - so I have to run undelete applications to recover files.  Any help would be appreciated ... Dupeguru is great

**Comment by butter-fly** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
